### DISCUSSION TITLE
twitter spaces recap - meeting #1 - 1.28.23

### DISCUSSION THREAD

**Discussion by tosufever** [veryengineer/collab#6] [General]
cool spaces. i’ll try to attend the next one. as penance for not participating, i took some notes & wrote a recap (this time). my knowledge is limited, so everyone should be able to edit my mistakes, or include something i missed. ideally a similar document could serve as a working document between spaces, and/or provide lazy shibes a tldr version of what was discussed.

much minutes
1.28.23 twitter spaces

    • wtf is going on? 
        ◦ this was the first in depth technical node conversation held on twitter spaces. maybe ever. much wow
        ◦ the networking is shrinking. node loss is a concern. Discussion
            ▪ Current node count is 705*
                • *for the record, this number should be 704. it is notably +1 within the last hour 
            ▪ there is a learning curve to running a full node to support the network 
                • even for moderately technically competent shibes
            ▪ some node loss is observed during version upgrades
            ▪ node loss from the 21’ social media campaign is particularly high
                • many shibes are using old pc’s and laptops to run full nodes on winblows
                    ◦ software signing ongoing issue
                    ◦ many older windows 10 machines will not migrate to 11
                    ◦ tpm hardware requirements increasingly problematic
            ▪ community driven edge nodes need to be easier & stickier
        ◦ many shibes are using old pc’s and laptops to run full nodes simply to support the network.
            ▪ using this type of old gear probably consumes 100+watts. an appropriate development board could do the same with 10watts (and likely have some room for future features or services)
        ◦ what shibes?
            ▪ There are a whole bunch of independently talented shibes who want to help support the network, or get involved, but they are all running in parallel. many working on similar independent projects or research. a more user friendly method of collaboration is needed
			
			
    • wtf are we trying to do?
        ◦ colaborate the shibes
            ▪ https://very.engineer 
            ▪ https://github.com/veryengineer
        ◦ build a box – literally. easily replicatable with simple blueprint like instructions and common hardware. keep it simple stupid
            ▪ https://github.com/veryengineer/collab
            ▪ it needs to be easier for shibes to deploy and maintain a node to support the network.
            ▪ discussion – what does that look like
                • [dev board] + [storage] + [flash based oobe installer] + [package verification]
                    ◦ plus a 3d printed case & a semi savvy shibe 
                • it should cost less than 1500 doge
                • shibes should be able to recreate the process easily using blueprints
                    ◦ eventually assembly could become a side hustle
                • future capabilities should consider
                    ◦ the addition of mining capabilities (probably not at start)
                        ▪ much complex
                    ◦ the addition of a renewable source of energy (ie solar panel & battery) easily built & assembled with the blueprints. 
                • choosing the appropriate dev board is critical
                    ◦ hardware availability – supply chain issues suck
                    ◦ long term support is important – fewer versions to maintain.
                • Software considerations discussed
                    ◦ image flash deployment prefered – flash to oobe “out of box experience”
                        ▪ post install software will need some verification pricedure
                    ◦ everyone agrees a virtualization layer between the board & software is ideal
                        ▪ probably lxc or docker
                    ◦ software direction partially dependent on dev board discussions 
                        ▪ it will probably be debian based. it’s also likely to be an arm processor? 
									
				
    • wtf needs focusing on? and probably whotf is going to focus on it?
        ◦ somehow all the cool stuff from discussion(s) needs to make it’s way into the collaborative space. 
            ▪ Actionable item: pat – collaborative space & wiki
        ◦ dev boards need to further discussed in detail & some decision on initial hardware support needs made.
            ▪ actionable item: osufever – create a top level thread for dev board discussion (include technical system requirements discussed here) 
            ▪ actionable item: everyone – participate in discussion(s)
            ▪ looks like a few specific topics were already created? Is there a way to categorize discussions? 
        ◦ we need to aggrigate links for similar projects
            ▪ actionable item: where do these go? - discussion started already.
        ◦ A list of who has what hardware that might be available for testing? 
            ▪ Maybe an inventory of sorts? 
			
			
     • wtf needs to happen before the next meeting?
        ◦ Schedule a time for the next meeting?
        ◦ Get more shibes involved
        ◦ Continued discussion on dev board hardware
            ▪ define a rough working scope?
        ◦ aggregate colaborative space & tools

    • wtf needs to happen during the next meeting?
        ◦ tbd   

[very.engineer.recap1.28.txt](https://github.com/veryengineer/collab/files/10561942/very.engineer.recap1.28.txt)

**Comment by patricklodder** [comment]
Thank you!!! ❤️
